src/widget/widget.h

```diff
@@ -104,6 +104,7 @@ class Widget final : public QMainWindow
     void searchCircle(CircleWidget* circleWidget);
     void searchItem(GenericChatItemWidget* chatItem, GenericChatItemWidget::ItemType type);
     bool groupsVisible() const;
+    void resetIcon();
```

Comment:
@zetok to reset icon status :)

I can't set value of `eventIcon` and `eventFlag` without it